scripts/teardown.sh

```diff
@@ -34,7 +35,7 @@ source "$ROOT"/scripts/common.sh
 # Otherwise we will run into the following error
 # "google_bigquery_dataset.gke-bigquery-dataset: googleapi:
 # Error 400: Dataset pso-helmsman-cicd-infra:gke_logs_dataset is still in use, resourceInUse"
-bq rm -r -f "${PROJECT}":"${BQ_LOG_DS}"
+bq rm -r -f --project_id="${PROJECT}" "${PROJECT}":"${BQ_LOG_DS}"
```

Comment:
I think that when the bq command is first run it brings up a prompt
```
Welcome to BigQuery! This script will walk you through the
process of initializing your .bigqueryrc configuration file.

First, we need to set up your credentials if they do not
already exist.

Credential creation complete. Now we will select a default project.
```

Which is still stopping us can you try changing the command to:

```bq rm -r -f --application_default_credential_file="${GOOGLE_APPLICATION_CREDENTIALS}" --project_id="${PROJECT}" "${PROJECT}":"${BQ_LOG_DS}"```

Reply:
Thanks bob will try that